task8/TutterApp/src/main/java/servlets/PostServlet.java

```diff
@@ -0,0 +1,67 @@
+package servlets;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dao.PostDAO;
+import dao.PostDAOImpl;
+import models.Constants;
+import models.Post;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Optional;
+
+@WebServlet("/post")
+public class PostServlet extends HttpServlet {
+    private PostDAO postDAO = new PostDAOImpl();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        Long id = Long.parseLong(req.getParameter("id"));
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Think of  extracting the duplicated code to an utility method or event utility class
You have had
```        
ObjectMapper objectMapper = new ObjectMapper();
objectMapper.setDateFormat(new SimpleDateFormat(Constants.DATE_FORMAT));
```
thrise is this class only, and several times elsewhere